py/foundation/entity.py

```diff
@@ -114,10 +113,8 @@ class Word(Entity):
   origin: Optional[InputWord] = None
 
   @staticmethod
-  def from_inputword(origin: InputWord) -> Optional['Word']:
-    text = origin.text
-    if text is None:
-      return None
+  def from_inputword(origin: InputWord) -> 'Word':
+    text = origin.text or ''
     return Word(text, origin.bounding_box, origin)
```

Comment:
I forget where we arrived on this conversation, can a `Word` take in a `None` as text?

Reply:
I think the main reason for closing https://github.com/instabase/foundation/pull/25 was more around whether or not all entities should have a text field. In the case of `Word`'s text field, I think we should assume that it won't be none?

Reply:
Ah got it, thanks!